packages/foundry/README.md

```diff
@@ -3,62 +3,66 @@
 A CLI to create new Workleap projects non-interactively. See `@workleap/foundry --help`:
 
 ```cmd
-Usage: foundry [options] [command]
+Usage: @workleap/foundry [options] [command]
 
 Foundry-CLI generator module
 
 Options:
-  -V, --version               output the version number
-  -h, --help                  display help for command
+  -V, --version                        output the version number
```

Comment:
It's a default `Commander` lib option. It can be change pretty easily.

I remember having read somewhere that they use `-V` because `-v` is for verbose...

But even for me it's strange to use the capital `V` for version. I will change this behavior.